awx/ui/src/components/JobList/JobListItem.test.js

```diff
@@ -51,7 +60,11 @@ describe('<JobListItem />', () => {
 
   test('initially renders successfully', () => {
     expect(wrapper.find('JobListItem').length).toBe(1);
+  });
```

Comment:
Do we assert anywhere that the Scheduled value does not show up when it shouldn't, if not we probably should.

Reply:
👍 Added a new test that checks for this case "should not show schedule detail in expanded view".